jpo-ode-consumer-example/src/main/java/com/consumerexample/app/ConsumerExample.java

```diff
@@ -0,0 +1,96 @@
+package com.consumerexample.app;
+
+import java.util.Properties;
+import java.util.Arrays;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.commons.cli.*;
+import us.dot.its.jpo.ode.plugin.j2735.J2735Bsm;
+import us.dot.its.jpo.ode.util.SerializationUtils;
+
+public class ConsumerExample {
+    
+    public static void main( String[] args ){
+    	
+    	// Option parsing
+    	Options options = new Options();
+    	
+    	Option bootstrap_server = new Option("b", "bootstrap-server", true, "Endpoint ('ip:port')");
+    	bootstrap_server.setRequired(true);
+    	options.addOption(bootstrap_server);
+
+    	Option topic_option = new Option("t", "topic", true, "Topic Name");
+    	topic_option.setRequired(true);
+    	options.addOption(topic_option);
+
+    	Option group_option = new Option("g", "group", true, "Consumer Group");
+    	group_option.setRequired(true);
+    	options.addOption(group_option);
+
+    	Option type_option = new Option("type", "type", true, "string|byte message type");
+    	type_option.setRequired(true);
+    	options.addOption(type_option);
+
+    	CommandLineParser parser = new DefaultParser();
+        HelpFormatter formatter = new HelpFormatter();
```

Comment:
Looks like indentation is a bit off in the nest 4 or so lines. Also, there is a mix of tabs and spaces in the files. Please check all the files to make sure only spaces are used.